types/scte224v20200407/viewingpolicy.go

```diff
@@ -0,0 +1,101 @@
+package scte224v20200407
+
+import (
+	"encoding/xml"
+	"fmt"
+	"time"
+
+	"github.com/Comcast/scte224structs/convert"
+	scte224_2015 "github.com/Comcast/scte224structs/types/scte224v20151115"
+	scte224_2018 "github.com/Comcast/scte224structs/types/scte224v20180501"
+)
+
+//Table 12
+type ViewingPolicy struct {
```

Comment:
got too cumbersome in one big file?

Reply:
Yes, was getting bigger with the new methods